dbfit-java/core/src/main/java/dbfit/fixture/RowSetFixture.java

```diff
@@ -64,7 +64,7 @@ private int findColumn(String name) throws Exception {
     }
 
     @Override
-    protected void bind(Parse heads) {
+    protected void bindColumnHeadersToMethodsAndFields(Parse heads) {
```

Comment:
The method in the parent `ColumnFixture` got renamed unclebob/fitnesse/issues/938